src/sidebar/components/test/excerpt-test.js

```diff
@@ -134,18 +134,18 @@ describe('Excerpt', () => {
       assert.equal(getExcerptHeight(wrapper), 200);
     });
 
-    it("sets button's default state to unpressed", () => {
+    it("sets button's default state to un-expanded", () => {
       const wrapper = createExcerpt({ inlineControls: true }, TALL_DIV);
       const button = wrapper.find('.excerpt__toggle-button');
-      assert.equal(button.prop('aria-pressed'), 'false');
+      assert.equal(button.prop('aria-expanded'), false);
```

Comment:
I think its fine to call out these in tests, but going forward, I think we can lean on axe to assert if we're OKAY or not in this capacity.  Thoughts?